.github/workflows/bors.yml

```diff
@@ -35,7 +35,7 @@ jobs:
       - uses: actions/checkout@v2
 
       - name: install Python
-        if: ${{ ! 1 }}
+        if: ${{ 'self-hosted' == 'ubuntu-latest' }}
```

Comment:
Is this comparing 2 strings that are always different?

Reply:
Yep, see [my other comment](https://github.com/leanprover-community/mathlib/pull/9198#discussion_r708336045).